SUMMARY.md

```diff
@@ -129,6 +130,7 @@
   - [UI Shell](libs/application/ui-shell/README.md)
 - [Auth API Lib](libs/auth-api-lib/README.md)
 - [Auth Nest Tools](libs/auth-nest-tools/README.md)
+  - [Rsk](libs/clients/rsk/README.md)
```

Comment:
you will need to create a README at the root for `lib/clients` so gitbook will be able to create the root navigation https://docs.devland.is/handbook/repository/gitbook-contribute#root-readme-md-withing-handbook-directory

Reply:
Fixed